xrootd/protocol/signRequirements.go

```diff
@@ -0,0 +1,59 @@
+// Copyright 2018 The go-hep Authors.  All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// Package protocol contains the XRootD protocol specific types.
```

Comment:
please remove this package-level comment as it may override the much nicer one that already exists in `protocol.go`.